### PR TITLE
changes for ACR 3.0.0

### PR DESCRIPTION
**Pull request by rosanch**
This commit contains changes necessary for the azure-arm-containerregistry 3.0.0

### CODE REVIEW

File: commands/azureCommands/run-buildTask.ts [hunk 1]
```diff
@@ -1,16 +1,16 @@
-import { BuildTaskBuildRequest } from "azure-arm-containerregistry/lib/models";
```
Comment (northtyphoon) [inline commands/azureCommands/run-buildTask.ts]: You may change the file name from `run-buildTask` to `run-Task`

File: commands/azureCommands/show-buildTask.ts [hunk 2]
```diff
@@ -1,43 +1,45 @@
 import { Registry } from "azure-arm-containerregistry/lib/models";
```
Comment (northtyphoon) [inline commands/azureCommands/show-buildTask.ts]: `show-buildTask` to `show-task`

File: explorer/models/taskNode.ts [hunk 3]
```diff
@@ -31,35 +31,35 @@ export class TaskRootNode extends NodeBase {
...
+        const taskNodes: TaskNode[] = [];
+        let tasks: ContainerModels.Task[] = [];
         const client = AzureUtilityManager.getInstance().getContainerRegistryManagementClient(element.subscription);
         const resourceGroup: string = acrTools.getResourceGroupName(element.registry);
-        buildTasks = await client.buildTasks.list(resourceGroup, element.registry.name);
-        if (buildTasks.length === 0) {
-            vscode.window.showInformationMessage(`You do not have any Build Tasks in the registry, '${element.registry.name}'. You can create one with ACR Build. `, "Learn More").then(val => {
+        tasks = await client.tasks.list(resourceGroup, element.registry.name);
+        if (tasks.length === 0) {
+            vscode.window.showInformationMessage(`You do not have any Tasks in the registry, '${element.registry.name}'. You can create one with ACR Build. `, "Learn More").then(val => {
```
Comment (northtyphoon) [inline explorer/models/taskNode.ts]: >Build [](start = 149, length = 5)

Task

File: explorer/models/taskNode.ts [hunk 4]
```diff
@@ -31,35 +31,35 @@ export class TaskRootNode extends NodeBase {
...
                 if (val === "Learn More") {
                     opn('https://aka.ms/acr/buildtask');
```
Comment (northtyphoon) [inline explorer/models/taskNode.ts]: >https://aka.ms/acr/buildtask [](start = 25, length = 28)

I create a new short link you can change to '[http://aka.ms/acr/task](http://aka.ms/acr/task)'. It's pointing to the old doc now I will redirect to to the task document once it's online.

File: commands/azureCommands/run-buildTask.ts (hunk 1 above)
Comment (rosanch) [reply commands/azureCommands/run-buildTask.ts]: I will make all the refactoring within another PR. In here are only functional changes, and minor refactoring changes.

**Pull request merged**